rnd/market/market/routes/admin.py

```diff
@@ -77,9 +77,8 @@ async def unset_agent_featured(
     A basic endpoint to unset an agent as featured in the database.
     """
     try:
-        await market.db.set_agent_featured(
-            agent_id, is_featured=False, category=category
-        )
+
+        await market.db.remove_featured_category(agent_id, category=category)
         return fastapi.responses.Response(status_code=200)
     except market.db.AgentQueryError as e:
         raise fastapi.HTTPException(status_code=500, detail=str(e))
```

Comment:
Instead of doing manually identify the AgentQueryError and add this in every route, isn't easier to just capture all the error message like `handle_internal_error` in server.py ?

Reply:
Technically we can let it raise itself lol. I’m following the pattern we have now but agreed there are better ways